js/app.js

```diff
@@ -42,18 +55,10 @@ addBtn.addEventListener('submit', (e) => {
   const titleInput = bookTitle.value;
   const authorInput = bookAuthor.value;
 
-  addBook(titleInput, authorInput);
+  Book.addBook(titleInput, authorInput);
 });
 
-function removeBook(index) {
-  const data = localStorage.booksList;
-  booksArray = JSON.parse(data);
-  booksArray.splice(index, 1);
-  localStorage.setItem('booksList', JSON.stringify(booksArray));
-  displayBooks();
-}
-
 window.addEventListener('load', () => {
-  displayBooks();
-  removeBook(id);
+  Book.displayBooks();
+  Book.removeBook(id);
```

Comment:
- [ ] good job refactoring your code 👍 however every time when the user reloads the page a book gets deleted as well, it looks like that's due to this part of the code.

Reply:
This issue has been well addressed. Nice job! :clap: :clap: